src/test/lib/operators/join_hash/join_hash_types_test.cpp

```diff
@@ -33,17 +33,13 @@ void test_hash_map(const std::vector<T>& values) {
 
   const auto& first_hash_map = hash_maps.at(0).value();
 
-  size_t row_count = 0;
-  for (const auto& pos_list_pair : first_hash_map) {
-    row_count += pos_list_pair.size();
-  }
-  EXPECT_EQ(row_count, partition.elements.size());
```

Comment:
Removed this part which was introspecting the details of the data structure too much for my taste.